### ISSUE TITLE
Loading animation

### ISSUE THREAD

**Issue by Pankajisyadav** [fast4x/RiMusic#2292]
### Description

I see this :-
![Screenshot_2024-06-14-15-19-32-828_it fast4x rimusic](https://github.com/fast4x/RiMusic/assets/107134678/a582ae0f-97ba-42b8-8e9f-da0e478e9920)
whenever I toggle from settings to quick picks for few seconds

### Suggested Solution

I think it would be better if a loading animation would be added instead of just plain text. Just my opinion.

### Additional Context

The app is good anyway

**Comment by ikanakova** [comment]
What is the animation? Like instead of "Sorry, tips are not available for now."? If so, I think it's useless, because then there will be tips anyway. Doesn't RiMusic really need to increase the size of the apk just to animate each temporary text...

**Comment by Pankajisyadav** [comment]
Yeah I think that would be better to show text only if the tips don't load after some time or if there's a connectivity issue.

**Comment by fast4x** [comment]
I can add a loader that is already in the app
